packages/node_modules/ciscospark/src/plugins/memberships.js

```diff
@@ -6,22 +6,27 @@ import {SparkPlugin, Page} from '@ciscospark/spark-core';
 
 /**
  * @typedef {Object} MembershipObject
- * @property {uuid} id - Unique identifier for the membership
+ * @property {string} id - Unique identifier for the membership
  * @property {string} roomId - The room ID
- * @property {uuid} personId - The person ID
+ * @property {string} personId - The person ID
  * @property {email} personEmail - The email address of the person / room member
- * @property {boolean} isModerator - Indicates whether the specified person should be a room moderator.
- * @property {boolean} isMonitor - Indicates whether the specified member is a room monitor.
- * @property {isoDate} created - The date and time that this membership was created.
+ * @property {boolean} isModerator - Indicates whether the specified person should be a room moderator
+ * @property {boolean} isMonitor - Indicates whether the specified member is a room monitor
+ * @property {isoDate} created - The date and time that this membership was created
```

Comment:
We should typedef isoDate somewhere. and maybe come up with better capitalization :)

Reply:
This has been addressed.